.github/workflows/trivy-analysis.yml

```
@@ -10,26 +10,36 @@ defaults:
 jobs:
 
   analysis:
-    name: Vulnerability scanner for nightly containers
+    name: Vulnerability scanner for containers
     runs-on: ubuntu-latest
     if: github.repository == 'keycloak/keycloak'
     strategy:
       matrix:
         container: [keycloak, keycloak-operator]
       fail-fast: false
     steps:
+      - name: Extract release ID
+        id: release
+        run: echo "id=${GITHUB_REF#refs/heads/release/}" >> $GITHUB_OUTPUT
+        
+      - name: Checkout code
+        uses: actions/checkout@v3
```

Why not use the latest version here?
```suggestion
        uses: actions/checkout@v4
```